### PR TITLE
This adds a profile page where users can view their points, favorited notes, and uploaded notes

### LINKED ISSUE #13: View Profile page
Load up the profile based on the user data in the database. Profile for _username_ can be found on /_username_.

- [x] Return User object on GET request from user servlet (Aradhya)
- [x] Take current mock and modify assuming you have a user object.

### PR DESCRIPTION
**Pull request by Aradhya-B**
# If Signed In
image
# If Not Signed In
image

TODO: Route between pages using Vue Router introduced by @andres-holguin  in #83

### CODE REVIEW

File: src/main/webapp/views/ProfilePage.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-app>
+    <navbar @setuser="onSetUser" @signedin="onSignIn"></navbar>
+    <div v-if="!signedIn">
+      <h1>Please sign in</h1>
+    </div> 
+    <div v-else> 
+        <profile-header :points="user.points"
```
Comment (keelywan) [inline src/main/webapp/views/ProfilePage.vue]: nit: indentation (I think it's indented twice) and same thing for below

**Review by andres-holguin — APPROVED** [review]
Very Nice :)

**Review by keelywan — APPROVED** [review]
Looks good!

File: src/main/webapp/views/ProfilePage.vue (hunk 1 above)
Comment (Aradhya-B) [reply src/main/webapp/views/ProfilePage.vue]: Ah you're right, good catch!

**Review by nickKelman — APPROVED** [review]

**Review by cftorres — APPROVED** [review]

**Pull request merged**